frontend/templates/product-list/SecondaryNavigation.tsx

```diff
@@ -42,12 +53,14 @@ export function SecondaryNavigation({ productList }: SecondaryNavigationProps) {
                   boxSizing="border-box"
                   justify="space-between"
                >
-                  <ProductListBreadcrumb
+                  <BreadCrumbs
                      display={{
                         base: 'none',
                         sm: 'flex',
                      }}
-                     productList={productList}
+                     breadCrumbs={breadCrumbs}
+                     breadcrumbsToShow={3}
```

Comment:
Minor: I think we may be comfortable allowing the breadcrumbs to be dynamic (size to the available space)

Reply:
Fine with me